File: packages/boozsoft-auth/src/main/java/org/boozsoft/service/AccountAccvoucherCdigestClassService.java
package org.boozsoft.service;

import org.boozsoft.domain.entity.account.AccountAccvoucherCdigestClass;
import org.springbooz.core.tool.result.R;
import org.springframework.web.bind.annotation.RequestParam;
import reactor.core.publisher.Mono;
import reactor.core.publisher.Flux;

import org.springframework.data.domain.Pageable;

public interface AccountAccvoucherCdigestClassService {
    Flux<AccountAccvoucherCdigestClass> findAll(Pageable pageable);

    Mono<AccountAccvoucherCdigestClass> findById(String id);

    Mono<AccountAccvoucherCdigestClass> save(AccountAccvoucherCdigestClass entity);

    Mono<Void> deleteById(String id);
}

File: docs/modules/boozsoft-server/src/main/kotlin/org/boozsoft/service/AccountAccvoucherCdigestClassService.kt
package org.boozsoft.service

import org.boozsoft.domain.entity.account.AccountAccvoucherCdigestClass
import org.springframework.data.domain.Pageable
import reactor.core.publisher.Flux
import reactor.core.publisher.Mono

interface AccountAccvoucherCdigestClassService {
    fun findAll(pageable: Pageable?): Flux<AccountAccvoucherCdigestClass?>?
    fun findById(id: String?): Mono<AccountAccvoucherCdigestClass?>?
    fun save(entity: AccountAccvoucherCdigestClass): Mono<AccountAccvoucherCdigestClass?>?
    fun deleteById(id: String?): Mono<Void?>?
}